src/chains/entities/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './backbone.entity';
+export * from './chain.entity';
+export * from './page.entity';
```

Comment:
What is the benefit of doing it like this?

Reply:
I'd say it's more concise to import the entities in multiple places this way:
```javascript
import { Backbone, Chain, Page } from './entities';
```
Another way to do it would be managing the exported interfaces:
```javascript
export { Backbone } from './backbone.entity';
export { Chain } from './chain.entity';
export { Page } from './page.entity';
```
I don't have a strong opinion on wether using `*` or not, it depends of the level of control you want to have over the exports. But doing it the manual way implies you'll need to edit the index if you plan to export several interfaces from the same file.

Reply:
Alright 👍 I'm going to comment on where we can use this in this PR (I see that some imports can then be shortened).